lib/generators/react_on_rails/templates/no_redux/base/client/app/bundles/HelloWorld/components/HelloWorldWidget.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+
+// Simple example of a React "dumb" component
+export default class HelloWorldWidget extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    // Uses lodash to bind all methods to the context of the object instance, otherwise
+    // the methods defined here would not refer to the component's class, not the component
+    // instance itself.
+    _.bindAll(this, '_handleChange');
+  }
+
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    updateName: PropTypes.func.isRequired,
+  };
+
+  // React will automatically provide us with the event `e`
+  _handleChange(e) {
+    const name = e.target.value;
+    this.props.updateName(name);
+  }
+
+  render() {
+    const _handleChange = this._handleChange;
```

Comment:
if there's only one dot and not too long and only used on time, then inline, IMHO